manifests/base/gitops-service-argocd/base/argo-cd-application-controller-clusterrolebinding.yaml

```diff
@@ -1,12 +1,15 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: appstudio-gitops-service-argocd-argocd-application-controller
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: appstudio-gitops-service-argocd-argocd-application-controller
-subjects:
-  - kind: ServiceAccount
-    name: gitops-service-argocd-argocd-application-controller
-    namespace: gitops-service-argocd
\ No newline at end of file
+
+# To enable 'gitops-service-argocd' instance as cluster-scoped Argo CD, uncomment the lines below.
+
```

Comment:
Can we also remove the ClusterRoles since they will not be used with the namespace-scoped instance? We can either comment the resource or its reference in the kustomization.yaml

 https://github.com/redhat-appstudio/managed-gitops/blob/main/manifests/base/gitops-service-argocd/base/kustomization.yaml

Reply:
Done, thanks @chetan-rns!